examples/_clip_driver/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .moss_clip import MossCLIP
+from .zero_shot_image_classifier import DataLake, ZeroShotImageClassifier, register_pipline
```

Comment:
This seems to reference modules that are not in this repo, is this the part you say can be removed before merging?